styles/landing.module.scss

```diff
@@ -0,0 +1,127 @@
+$committee-color: #5eda12;
+
+.masthead {
+  background-image: url("../public/Illustration.png");
```

Comment:
Can we delete the crop image from our project since we are using the png instead